src/index.html

```diff
@@ -597,17 +597,17 @@ <h2 class="title__heading">
               <i class="contact__icon-inner icon-phone"></i>
             </div>
             <div class="contact__info">
-              +8801743331996<br>
-              +8801928737807
+              <a href="tel:+8801743331996">+8801743331996</a>
+              <a href="tel:+8801928737807">+8801928737807</a>
```

Comment:
Mmm, maybe `+7`?

Reply:
And it's better to format them somehow, for example: `+7(801)743-31-99` 
Hm, too much digits, maybe it's not `+8` but `+880`.